src/color/creating_reading.js

```diff
@@ -288,25 +288,12 @@ p5.prototype.brightness = function(c) {
  * @param  {Number}        [alpha]
  * @return {p5.Color}
  */
-
 /**
  * @method color
- * @param  {String}        value   a color string
- * @param  {Number}        [alpha]
- * @return {p5.Color}
- */
-/**
- * @method color
- * @param  {Number[]}      values  an array containing the red,green,blue &
- *                                 and alpha components of the color
- * @return {p5.Color}
- */
-/**
- * @method color
- * @param  {p5.Color}     color
+ * @param {p5.Color|Number[]|String} color p5.Color object, color components,
```

Comment:
for example this one would remove the three independent bottom signatures in the image, replacing it with one single line siganture. unless I am missing something?

Reply:
indeed, but those are all just different types for representing the same thing - a color. and this is not new, it's just borrowed from most of the other color-taking methods, for example, see the existing `brightness`: https://p5js.org/reference/#/p5/brightness. there are a bunch that are like this. this PR just makes all the color-taking methods look the same, and fixes a bunch of the errors.

there are many instances of methods whose parameters can be of different types. for examples: https://p5js.org/reference/#/p5/image the `img` parameter can be either a `p5.Image` or a `p5.Element`. the docs have always been like this and the validation code that uses them has handled these type alternations since when i first looked at it.

Reply:
in this case we felt it was important because color takes so many different formats that it can become very confusing. the priority here is making things clear for the beginning programmer. i'm pretty set on this, i'm sorry.

Reply:
ok, are you saying that you want `color` to have different docs than all the other methods that take the same parameters?

Reply:
if there are other functions that are as confusing as color, i am open to breaking some of those out as well

Reply:
there's a bunch. basically anything that calls `p5.prototype.color()`, `this.color()` or `this._pInst.color.apply()`.

i think pointLight will require 24 overloads.

Reply:
I feel like you are trying to pick a fight and I'm not interested in that. I'm leaving this thread now.

Reply:
oh no not at all, i'm sorry you got that impression. i'm just trying to work out which methods you want to have these overload separated, and which ones not. i'd like to get the errors in the docs fixed because they are causing errors in the mocha tests that i'm trying to fix.